packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

```diff
@@ -283,7 +283,8 @@ export default VSelect.extend({
 
       input.data = input.data || {}
       input.data.attrs = input.data.attrs || {}
-      input.data.attrs.autocomplete = input.data.attrs.autocomplete || 'disabled'
+      input.data.attrs.readonly = true
```

Comment:
That makes autocomplete and combobox readonly

Reply:
I didn't think of that :man_facepalming: